lessons/de/chapter_5.yaml

```diff
@@ -0,0 +1,224 @@
+- title: Kapitel 5 - Eigentums- und Kreditdaten
+  content_markdown: >
+    Rust verfügt im Vergleich zu anderen Programmen über ein einzigartiges Paradigma für die
+    Speicherverwaltung Sprachen. Wir werden uns das ansehen
+
+    die Verhaltensweisen und Validierungen des Compilers nacheinander, also nicht
+    überwältigend. Es ist wichtig, sich daran zu erinnern 
+
+    Letztlich dienen die Regeln, die wir aufzeigen, nicht dazu, Ihnen das Leben schwer zu machen, sondern um Ihnen zu helfen
+    Sie machen Ihren Code weniger fehleranfällig!
+- title: Eigentum
+  content_markdown: >
+    Durch die Instanziierung eines Typs und dessen **Bindung** an einen Variablennamen wird ein Speicher erstellt
+    Ressource, die der Rust-Compiler validiert
+    
+    durch sein ganzes **Leben**. Die gebundene Variable wird als Ressource bezeichnet
+    **Eigentümer**.
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=struct%20Foo%20%7B%0A%20%20%20%20x%3A%20i32%2C%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20%2F%2F%20Instan%C8%9Biem%20structuri%20(Foo)%20%C8%99i%20le%20leg%C4%83m%20de%20variabile%20(foo)%0A%20%20%20%20%2F%2F%20pentru%20a%20crea%20resurse%20%C3%AEn%20memorie%0A%20%20%20%20let%20foo%20%3D%20Foo%20%7B%20x%3A%2042%20%7D%3B%0A%20%20%20%20%2F%2F%20foo%20este%20proprietarul%0A%7D%0A
```

Comment:
Please translate the code to German or use the English version instead of the Romanian version.